samples/angular-app/src/api/endpoints/pets/pets.msw.ts

```diff
@@ -7,11 +7,13 @@
 import {
   rest
 } from 'msw'
-import faker from 'faker'
+import {
+  faker
+} from '@faker-js/faker'
 
-export const getListPetsMock = () => ([...Array(faker.datatype.number({min: 1, max: 10}))].map(() => ({id: faker.datatype.number(), name: (()=>faker.name.lastName())(), tag: (()=>faker.name.lastName())()})))
+export const getListPetsMock = () => ([...Array(faker.datatype.number({min: 1, max: 10}))].map(() => ({id: faker.datatype.number(), name: (()=>import_faker.faker.name.lastName())(), tag: (()=>import_faker.faker.name.lastName())()})))
```

Comment:
For some reason custom defined mocks from `orval.config.ts` are getting prepended with `import_faker` and break. I can't get my head around why this is happening. 

It has to do something with the import.
```ts
import * as faker from '@faker-js/faker';

// would need this usage:
faker.faker.name.lastName()
```
Works, but is definitely not usable/intended.

The following imports don't work:
```ts
import faker from '@faker-js/faker';

// results in:
import_faker.default.name.lastName()
```
```ts
import { faker } from '@faker-js/faker';

// results in:
import_faker.faker.name.lastName()
```

`import { faker } from '@faker-js/faker';` is the recommended import by `@faker-js/faker`

@anymaniax @melloware Maybe one of you guys could poke me in the right direction. 🙏

Reply:
I will see if I see anything but @anymaniax is the expert here.

Reply:
OK I looked I am not quite sure why this is happening but I agree with you the correct import should be `mport { faker } from '@faker-js/faker';`  I just can't see what is making it do `import_faker`?

Reply:
it's because esbuild is reading the config and transforming the code. Didn't find a solution to avoid that. But you can replace the import in the string directly.

Reply:
You can replace the content of this [file](https://github.com/anymaniax/orval/blob/master/src/core/generators/mocks.ts) with this [code](https://gist.github.com/anymaniax/c206c2a98d7a476e7f9e0f1829c254e7)